js/package.json

```
@@ -5,13 +5,10 @@
   "main": "src/core.js",
   "license": "GPL-3.0",
   "scripts": {
-    "preinstall": "cd ../wasm && wasm-pack build --out-dir ../js/aleo-wasm --target web --scope entropy-1729",
-    "generate_docs": "jsdoc --configure jsdoc.json --verbose",
+    "preinstall": "cd ../wasm && wasm-pack build --out-dir ../js/aleo-wasm --target nodejs",
```

The `cd` could be removed

```suggestion
    "preinstall": "wasm-pack build wasm/ --out-dir ../js/aleo-wasm --target nodejs",
```

---

That's from the original aleo-sdk code from Aleo. This directory it's not going to be used at least for now. The suggestion is valid anyways so I committed the change.